src/dialog.css

```diff
@@ -0,0 +1,10 @@
+.dialog form {
+    width: 360px;
+}
+.h1 {
+    align-items: center;
+    justify-content: space-between;
+    display: flex;
+    flex-direction: row;
+}
+.label {display: flex}
```

Comment:
let's be consistent with how we write our code. This is really important. For this line, we should add a line break to have the property in a new line

```
.label {
  display: flex
}
```


Also, let's change to 2 spaces for indentation as that is another of our guidelines.


Some more info about our code conventions:

https://gitlab.com/SUSE-UIUX/eos/-/wikis/home#writing-code